### DISCUSSION TITLE
Vulnerabilities on Docker

### DISCUSSION THREAD

**Discussion by gregstack** [frePPLe/frepple#645] [Q&A]
Hello

I'm trying to run container on Docker concerning Frepple community edition but I met some issues about vulnerability. please could you give me the way to solve them ? Thank you for your help

![Capture d’écran 2024-04-15 à 12 11 42](https://github.com/frePPLe/frepple/assets/91198315/b5323da1-0dff-460c-97cc-edb0ba688c1c)

**Comment by jdetaeye** [comment]
This docker scan gives a lot of positives that are either low priority or not applicable to frepple.   

Reviewing the list:
- django 4.2.8:    frepple uses a customized version of django 4.2.8, see from https://github.com/frePPLe/django.  This can be easily updated to the match with 4.2.12.   The resolved issues are not really used in our code, but better be safe than sorry...
- setuptools and pip:    these dependencies aren't used during runtime.  No big deal.
- apache:   Used extensively by frepple.  Best to upgrade indeed.  Ubuntu package can be updated in your container by running "apt update && apt upgrade"
- remaining ones:   These are well beyond our codebase. Running "apt update && apt upgrade" will bring these up to date.

IMHO only the first one needs action.   The others are irrelevant, or will be automatically fixed in the next release (when the new release is based on a updated ubuntu base image).

**Reply by gregstack** [reply]
Thanks a lot for your responsiveness ! I'll check them
Best Regards
